waku.nimble

```diff
@@ -116,11 +92,24 @@ task chat2bridge, "Build chat2bridge":
   let name = "chat2bridge"
   buildBinary name, "apps/chat2bridge/", "-d:chronicles_log_level=TRACE"
 
-### Waku Tooling
-task wakucanary, "Build waku-canary tool":
-  let name = "wakucanary"
-  buildBinary name, "tools/wakucanary/", "-d:chronicles_log_level=TRACE"
 
-task networkmonitor, "Build network monitor tool":
-  let name = "networkmonitor"
-  buildBinary name, "tools/networkmonitor/", "-d:chronicles_log_level=TRACE"
+### Legacy: Whisper & Waku v1 tasks
```

Comment:
Shall we remove this ?

Reply:
We have not yet deprecated Waku v1. So I cannot remove legacy code and the associated tasks.